### DISCUSSION TITLE
p-value interpretation

### DISCUSSION THREAD

**Discussion by SamuelBeauregard** [nicholasjclark/mvgam#64] [Q&A] [answered]
What do p-values mean in mvgam's context? I saw somewhere (I can't find the source anymore) that bayesian p-values meant that the parameters that had a p-value lower than 0.05 explained the variations of the independent variable better than a null model. However, when talking to people more knowledgeable in bayesian statistics than I am, nobody seems convinced. I also noticed that there is no mentions of p-values in this paper. https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13974

I'm working on a model extremely similar to the one in question 62, but I don't get 'significant' p-values with the data of this question so I included the output of another similar model for this question's clarity (see below)  
https://github.com/nicholasjclark/mvgam/discussions/62

![image](https://github.com/user-attachments/assets/8fbdc614-e74c-4a66-958b-940642da37d0)

In this example, it would mean that the variation of Pseudomonas and Alkanindiges through time would explain better than a null model the variations of my genus of interest. If I'm not mistaken however, p-values are more a frequentist thing than a bayesian one so I'm assuming it's much less absolute than in a frequentist context. Therefore, would it be fair to still consider that the parameters that are not significant/have a p-value higher than 0.05 can still have an influence on the response variable (depending on the smooths and CI)? 

More precisely in my example, it could mean that the whole community (more than 600 genus) may have a deterministic, but diffuse influence on the variation of a specific genus. In this case, some genus would have a stronger influence (Pseudomonas and Alkanindiges as they're significant). Please note that that part merely is a hypothesis for further research, I don't intend on testing it.

I have yet to find colleagues that work with time series so I come here a lot. Your help is extremely appreciated!

**Comment by SamuelBeauregard** [comment]
I just found the discussion on p-values around 52:20, basically they're not very reliable
https://www.youtube.com/watch?v=0zZopLlomsQ&t=3495s

**Reply by nicholasjclark** [reply, marked as answer]
Yes generally I wouldn't rely on them too much as they are sensitive to the decision around how to calculate EDF and they ignore important uncertainties. But I elected to leave them in as they are often expected by mgcv users
